app/static/js/app.js

```diff
@@ -123,12 +125,11 @@ function sendKeystroke(keystroke) {
 function onSocketConnect() {
   connectedToServer = true;
   document.getElementById("connection-indicator").connected = true;
-  if (document.getElementById("shutdown-wait").show) {
-    location.reload();
-  }
+  restoreCursor();
 }
 
 function onSocketDisconnect(reason) {
+  setCursor(0, false);
```

Comment:
Why do we need to change the cursor here?

Reply:
if the server is disconnected it should show the disabled cursor.